server/datastore/mysql/hosts_test.go

```diff
@@ -1000,6 +1001,28 @@ func testHostsListMunkiIssueID(t *testing.T, ds *Datastore) {
 	require.ElementsMatch(t, []string{strings.Repeat("A", maxMunkiIssueNameLen), strings.Repeat("☺", maxMunkiIssueNameLen)}, names)
 }
 
+func testHostsListMissing(t *testing.T, ds *Datastore) {
+	for i := 0; i < 15; i++ {
+		_, err := ds.NewHost(context.Background(), &fleet.Host{
+			SeenTime:      time.Now().Add(-time.Duration(i) * time.Hour * 24),
+			OsqueryHostID: strconv.Itoa(i),
+			NodeKey:       fmt.Sprintf("%d", i),
+		})
+		require.NoError(t, err)
+		if err != nil {
+			return
+		}
+	}
```

Comment:
`require.NoError` will exit the test if there's an error!

```suggestion
		require.NoError(t, err)
	}
```

Reply:
Hah, thanks. I copied that from line testHostsListStatus and seems I didn't question it. I'll fix both.